src/modules/services-systemd/services-systemd.schema.yaml

```diff
@@ -4,22 +4,18 @@
 $schema: https://json-schema.org/schema#
 $id: https://calamares.io/schemas/services-systemd
 definitions:
-    service:
-        $id: 'definitions/service'
+    unit:
+        $id: 'definitions/unit'
         type: object
-        description: a name and a flag for services, targets, and others
+        description: a map containing a unit name, an action, and whether it is mandatory
```

Comment:
A minor point but I think in yaml it is called a "mapping", not a "map"

Reply:
Changed